From: Arnaud Ferraris <arnaud.ferraris@gmail.com>
Date: Fri, 4 Mar 2022 02:00:43 +0100
Subject: pine64_defconfig: enable AF8133 magnetometer

This device replaces the LIS3MDL found in previous PinePhone batches.
---
 arch/arm64/configs/pine64_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/pine64_defconfig b/arch/arm64/configs/pine64_defconfig
index aa73eca..0be2ac8 100644
--- a/arch/arm64/configs/pine64_defconfig
+++ b/arch/arm64/configs/pine64_defconfig
@@ -2006,6 +2006,7 @@ CONFIG_VEML6030=m
 CONFIG_VEML6070=m
 CONFIG_VL6180=m
 CONFIG_ZOPT2201=m
+CONFIG_AF8133J=m
 CONFIG_AK09911=m
 CONFIG_BMC150_MAGN_I2C=m
 CONFIG_BMC150_MAGN_SPI=m
